Mathlib/CategoryTheory/Enriched/Basic.lean

```diff
@@ -393,6 +393,91 @@ structure GradedNatTrans (A : Center V) (F G : EnrichedFunctor V C D) where
       (A.2.β (X ⟶[V] Y)).hom ≫ (F.map X Y ⊗ₘ app Y) ≫ eComp V _ _ _ =
         (app X ⊗ₘ G.map X Y) ≫ eComp V _ _ _
 
+attribute [reassoc] GradedNatTrans.naturality
+
+/-- A (non-graded) natural transformation between to enriched functors is a natural
+transformation on graded on `𝟙_ V`. -/
```

Comment:
I do not know where the term "graded natural transformation" (from `GradedNatTrans`) we are using comes from, which can be quite confusing. Do you know a reference which uses this terminology? and if so, could you add a reference?